Frontend/src/app/user-search-interface/user-search-interface.component.html

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
![image](https://user-images.githubusercontent.com/75213424/154852184-5891b873-f243-49ae-ab3e-8d4cf3671dfe.png)
why it centered to left size? It will have something in space?

Reply:
Mabye